Build fails - The following artifacts could not be resolved: com.smartbear.utils.analytics:analytics-core:jar:5.5.4-SNAPSHOT, com.smartbear.utils.analytics:out-app-analytics-provider:jar:5.5.4-SNAPSHOT

[ERROR] Failed to execute goal on project soapui: Could not resolve dependencies for project com.smartbear.soapui:soapui:jar:5.7.1-SNAPSHOT: The following artifacts could not be resolved: com.smartbear.utils.analytics:analytics-core:jar:5.5.4-SNAPSHOT, com.smartbear.utils.analytics:out-app-analytics-provider:jar:5.5.4-SNAPSHOT: Could not find artifact com.smartbear.utils.analytics:analytics-core:jar:5.5.4-SNAPSHOT in sonatype-snapshots (https://oss.sonatype.org/content/repositories/snapshots/) -> [Help 1]

--- a/pom.xml
+++ b/pom.xml
@@ -38,7 +38,7 @@
         <project.name.visible>SoapUI</project.name.visible>
         <project.src.artifactId>soapui</project.src.artifactId>
 
-        <analytics.version>5.5.4-SNAPSHOT</analytics.version>
+        <analytics.version>5.5.3</analytics.version>
 
         <groovy-all.version>3.0.6</groovy-all.version>
 

